staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -269,37 +309,56 @@ func (cfgCtlr *configController) processNextWorkItem() bool {
 
 	func(obj interface{}) {
 		defer cfgCtlr.configQueue.Done(obj)
-		if !cfgCtlr.syncOne() {
+		report := cfgCtlr.SyncOne(map[string]string{})
+		switch {
+		case report.NeedRetry:
 			cfgCtlr.configQueue.AddRateLimited(obj)
-		} else {
+		case report.NeededSpecificWait > 0:
+			cfgCtlr.configQueue.AddAfter(obj, report.NeededSpecificWait)
+		default:
 			cfgCtlr.configQueue.Forget(obj)
 		}
 	}(obj)
 
 	return true
 }
 
-// syncOne attempts to sync all the API Priority and Fairness config
-// objects.  It either succeeds and returns `true` or logs an error
-// and returns `false`.
-func (cfgCtlr *configController) syncOne() bool {
+// SyncReport describes the outcome of a call to SyncOne
+type SyncReport struct {
+	// Need to wait this much time for the sake of rate limiting
+	// controller fights
+	NeededSpecificWait time.Duration
+
+	// NeedRetry indicates whether to requeue with exponential backoff
+	NeedRetry bool
+}
+
+// SyncOne does one full synchronization.  It reads all the API
+// objects that configure API Priority and Fairness and updates the
+// local configController accordingly.
+// Only invoke this in the one and only worker goroutine
+func (cfgCtlr *configController) SyncOne(flowSchemaRVs map[string]string) SyncReport {
+	klog.V(5).Infof("%s SyncOne at %s", cfgCtlr.name, cfgCtlr.clock.Now().Format(timeFmt))
 	all := labels.Everything()
 	newPLs, err := cfgCtlr.plLister.List(all)
 	if err != nil {
 		klog.Errorf("Unable to list PriorityLevelConfiguration objects: %s", err.Error())
-		return false
+		return SyncReport{NeedRetry: true}
 	}
 	newFSs, err := cfgCtlr.fsLister.List(all)
 	if err != nil {
 		klog.Errorf("Unable to list FlowSchema objects: %s", err.Error())
-		return false
+		return SyncReport{NeedRetry: true}
 	}
-	err = cfgCtlr.digestConfigObjects(newPLs, newFSs)
+	suggestedDelay, err := cfgCtlr.digestConfigObjects(newPLs, newFSs, flowSchemaRVs)
+	//if suggestedDelay > 0 {
+	//	cfgCtlr.configQueue.AddAfter(0, suggestedDelay)
+	//}
```

Comment:
nit: remove

Reply:
Oh yes, that was an editing oversight.  Gone now.  Thanks.